outbox-kafka-spring/src/test/java/one/tomorrow/transactionaloutbox/service/OutboxProcessorIntegrationTest.java

```diff
@@ -267,7 +263,7 @@ public void should_ContinueProcessingAfterDbConnectionFailureInPreventLockSteali
         CountDownLatch cdl = new CountDownLatch(1);
 
         OutboxLockRepository failingLockRepository = (OutboxLockRepository) beanFactory.applyBeanPostProcessorsAfterInitialization(
-                new OutboxLockRepository(sessionFactory) {
+                new OutboxLockRepository() {
```

Comment:
Same here: this now needs to get an `entityManager`.

Reply:
I think this is because of the way the `OutboxProcessor` instantiates its fields. It works with the `DefaultOutboxEntityManager` still existing, that's why I undid my last commit that removed it.

I see two options:

1. Keep the `DefaultOutboxEntityManager` for now.
2. Remove it, but then also change the way the `OutboxProcessor` instantiates its fields.

I'd lean towards option 1, leaving option 2 as a mid-term goal we want to achieve, but not necessarily in this PR. Would that work for you or would option 2 be necessary for you to approve this PR?

Reply:
Maybe add a default constructor and a constructor accepting an `EntityManager` (to be used by this test here)?

Reply:
Good idea, that seems to work! I've added another commit, re-removing the `OutboxEntityManager` and adding the two constructors. On my local machine, all tests have passed, so I have good hopes for the pipeline here. 🙂

Reply:
@magro, @hnrkdmsk, @danielrehmann, anything missing from your side for approval and merge? 🙂

Reply:
LGTM now, thanks for all your efforts!